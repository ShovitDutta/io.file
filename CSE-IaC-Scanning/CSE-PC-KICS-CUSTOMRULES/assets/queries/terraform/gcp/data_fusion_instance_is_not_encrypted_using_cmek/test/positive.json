{
    "format_version": "1.2",
    "terraform_version": "1.7.5",
    "planned_values": {
        "root_module": {
            "resources": [
                {
                    "address": "google_data_fusion_instance.cmek",
                    "mode": "managed",
                    "type": "google_data_fusion_instance",
                    "name": "cmek",
                    "provider_name": "registry.terraform.io/hashicorp/google",
                    "schema_version": 0,
                    "values": {
                        "accelerators": [],
                        "crypto_key_config": [
                            {
                                "key_reference": "projects/cvs-key-vault-test/locations/us-east2/keyRings/test-nonprod-nam3/cryptoKeys/gk-hcb-nonprod-sde-dbaas-nam3"
                            }
                        ],
                        "dataproc_service_account": null,
                        "description": null,
                        "display_name": null,
                        "effective_labels": {
                            "goog-terraform-provisioned": "true"
                        },
                        "enable_rbac": null,
                        "enable_stackdriver_logging": true,
                        "enable_stackdriver_monitoring": true,
                        "event_publish_config": [],
                        "labels": null,
                        "name": "my-instance",
                        "network_config": [],
                        "private_instance": true,
                        "project": "security-controls-sandbox",
                        "region": "us-central1",
                        "terraform_labels": {
                            "goog-terraform-provisioned": "true"
                        },
                        "timeouts": null,
                        "type": "BASIC"
                    },
                    "sensitive_values": {
                        "accelerators": [],
                        "crypto_key_config": [
                            {}
                        ],
                        "effective_labels": {},
                        "event_publish_config": [],
                        "network_config": [],
                        "options": {},
                        "terraform_labels": {}
                    }
                }
            ]
        }
    },
    "resource_changes": [
        {
            "address": "google_data_fusion_instance.cmek",
            "mode": "managed",
            "type": "google_data_fusion_instance",
            "name": "cmek",
            "provider_name": "registry.terraform.io/hashicorp/google",
            "change": {
                "actions": [
                    "create"
                ],
                "before": null,
                "after": {
                    "accelerators": [],
                    "crypto_key_config": [
                        {
                            "key_reference": "projects/cvs-key-vault-test/locations/us-east2/keyRings/test-nonprod-nam3/cryptoKeys/gk-hcb-nonprod-sde-dbaas-nam3"
                        }
                    ],
                    "dataproc_service_account": null,
                    "description": null,
                    "display_name": null,
                    "effective_labels": {
                        "goog-terraform-provisioned": "true"
                    },
                    "enable_rbac": null,
                    "enable_stackdriver_logging": true,
                    "enable_stackdriver_monitoring": true,
                    "event_publish_config": [],
                    "labels": null,
                    "name": "my-instance",
                    "network_config": [],
                    "private_instance": true,
                    "project": "security-controls-sandbox",
                    "region": "us-central1",
                    "terraform_labels": {
                        "goog-terraform-provisioned": "true"
                    },
                    "timeouts": null,
                    "type": "BASIC"
                },
                "after_unknown": {
                    "accelerators": [],
                    "api_endpoint": true,
                    "create_time": true,
                    "crypto_key_config": [
                        {}
                    ],
                    "effective_labels": {},
                    "event_publish_config": [],
                    "gcs_bucket": true,
                    "id": true,
                    "network_config": [],
                    "options": true,
                    "p4_service_account": true,
                    "service_endpoint": true,
                    "state": true,
                    "state_message": true,
                    "tenant_project_id": true,
                    "terraform_labels": {},
                    "update_time": true,
                    "version": true,
                    "zone": true
                },
                "before_sensitive": false,
                "after_sensitive": {
                    "accelerators": [],
                    "crypto_key_config": [
                        {}
                    ],
                    "effective_labels": {},
                    "event_publish_config": [],
                    "network_config": [],
                    "options": {},
                    "terraform_labels": {}
                }
            }
        }
    ],
    "configuration": {
        "provider_config": {
            "google": {
                "name": "google",
                "full_name": "registry.terraform.io/hashicorp/google",
                "version_constraint": "6.15.0",
                "expressions": {
                    "project": {
                        "constant_value": "security-controls-sandbox"
                    },
                    "region": {
                        "constant_value": "us-central1"
                    },
                    "zone": {
                        "constant_value": "us-central1-c"
                    }
                }
            }
        },
        "root_module": {
            "resources": [
                {
                    "address": "google_data_fusion_instance.cmek",
                    "mode": "managed",
                    "type": "google_data_fusion_instance",
                    "name": "cmek",
                    "provider_config_key": "google",
                    "expressions": {
                        "crypto_key_config": [
                            {
                                "key_reference": {
                                    "constant_value": "projects/cvs-key-vault-test/locations/us-east2/keyRings/test-nonprod-nam3/cryptoKeys/gk-hcb-nonprod-sde-dbaas-nam3"
                                }
                            }
                        ],
                        "enable_stackdriver_logging": {
                            "constant_value": true
                        },
                        "enable_stackdriver_monitoring": {
                            "constant_value": true
                        },
                        "name": {
                            "constant_value": "my-instance"
                        },
                        "private_instance": {
                            "constant_value": true
                        },
                        "region": {
                            "constant_value": "us-central1"
                        },
                        "type": {
                            "constant_value": "BASIC"
                        }
                    },
                    "schema_version": 0
                }
            ]
        }
    },
    "timestamp": "2025-07-21T15:58:47Z",
    "errored": false
}