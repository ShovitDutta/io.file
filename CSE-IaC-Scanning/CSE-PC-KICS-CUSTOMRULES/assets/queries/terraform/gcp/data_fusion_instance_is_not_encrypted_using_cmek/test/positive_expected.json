[
    {
        "queryName": "data fusion instance is not encrypted using cmek",
        "severity": "HIGH",
        "line": 16
    }
]